pkg/templateservicebroker/servicebroker/servicebroker.go

```diff
@@ -55,6 +70,7 @@ func NewBroker(saKubeClientConfig *restclient.Config, informer templateinformer.
 		hasSynced:          informer.Informer().HasSynced,
 		templateNamespaces: templateNamespaces,
 		restmapper:         restutil.DefaultMultiRESTMapper(),
+		gcCreateDelay:      time.Duration(delay),
```

Comment:
should be `delay*time.Second` (although using time.ParseDuration might be preferable so the user can specify the unit)

Reply:
well i'm ultimately multiplying it by time.Second at the point of use but sure i guess that would be cleaner.

Reply:
missed that, sorry